tests/Unit/ProjectCodeTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright (c) 2013-2017 OpenCFP
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ *
+ * @see https://github.com/opencfp/opencfp
+ */
+
+namespace OpenCFP\Test\Unit;
+
+use Localheinz\Test\Util\Helper;
+use OpenCFP\Domain;
+
+use OpenCFP\Http;
+use OpenCFP\Provider;
+use PHPUnit\Framework;
+
+/**
+ * @coversNothing
+ */
+final class ProjectCodeTest extends Framework\TestCase
```

Comment:
I think we need a better name for this test class

Reply:
I'm open for suggestions, @BackEndTea!